BackofficeBundle/Resources/public/coffee/orchestraLib.coffee

```diff
@@ -29,12 +29,9 @@ opts =
   openedSign: "<em class=\"fa fa-minus-square-o\"></em>"
 $("#left-panel nav").data({opts : opts})
 
-displayMenu = (route) ->
-  selectedPath = undefined
-  if typeof route isnt "undefined"
-    selectedPath = "#" + route
-  else
-    selectedPath = "#" + Backbone.history.fragment
+displayMenu = (route, refresh) ->
+  selectedPath = "#" + (route || Backbone.history.fragment)
+  refresh = refresh || (typeof route == "undefined")
```

Comment:
will verify if route not null, won't work with undefined. tested @itkg-alavieille